tests/swap_test.go

```diff
@@ -149,7 +81,7 @@ var _ = Describe("[Serial][sig-compute]SwapTest", Serial, decorators.SigCompute,
 		Expect(len(nodes.Items)).To(BeNumerically(">", 1),
 			"should have at least two schedulable nodes in the cluster")
 
-		skipIfSwapOff(fmt.Sprintf("swap should be enabled through env var: KUBEVIRT_SWAP_ON=true "+
+		skipIfSwapOff(virtClient, fmt.Sprintf("swap should be enabled through env var: KUBEVIRT_SWAP_ON=true "+
```

Comment:
no need to pass virtClient you can just use kubevirt.Client() wherever you use it